lib/main.dart

```diff
@@ -94,8 +94,9 @@ class _MyAppState extends State<MyApp> {
       locale: locale,
       title: 'OpenEMR',
       theme: new ThemeData(
-        primarySwatch: Colors.red,
-      ),
+          primarySwatch: Colors.blue,
+          iconTheme:
+              IconThemeData(color: Colors.green, opacity: 1, size: 20.0)),
```

Comment:
Hey I think these iconsize should change according to device screen for responsiveness. Right now the content is overflowing in my mobile?

Reply:
I guess I mentioned before that I am not worried about the responsiveness yet. I am currently more interested in getting a tweakable UI with a bit of working functionality. I added that `iconthemedata` so one can get the idea of how Theme works.